core/src/main/kotlin/org/evomaster/core/output/service/ApiTestCaseWriter.kt

```diff
@@ -44,6 +45,10 @@ abstract class ApiTestCaseWriter : TestCaseWriter() {
         if (initializingMongoResults.any { (it as? MongoDbActionResult) == null })
             throw IllegalStateException("the type of results are expected as MongoDbActionResults")
 
+        val initializingHostnameResolutionActions = ind.individual
+            .seeInitializingActions()
+            .filterIsInstance<HostnameResolutionAction>()
+            .distinct()
```

Comment:
why `distinct`? they should be by construction distinct. if not, it is a bug in the handling of insertions into individual

Reply:
fixed it at the insertion, forget to remove this.